instasham/instasham/src/main/resources/application.properties

```diff
@@ -0,0 +1,9 @@
+spring.application.name=instasham
+spring.datasource.driver-class-name=com.mysql.jdbc.Driver
+spring.datasource.url = jdbc:mysql://localhost:port/databasename
```

Comment:
Is ``port`` a placeholder?  
I know we will eventually replace this with an env var but this should be a number right?